addons/sale/views/res_config_settings_views.xml

```diff
@@ -357,6 +357,20 @@
                                 </div>
                             </div>
                         </div>
+                        <div class="col-12 col-lg-6 o_setting_box" id="shipping_costs_shiprocket">
+                            <div class="o_setting_left_pane">
+                                <field name="module_delivery_shiprocket" widget="upgrade_boolean"/>
+                            </div>
+                            <div class="o_setting_right_pane">
+                                <label for="module_delivery_shiprocket"/>
+                                <div class="text-muted">
+                                    Compute shipping costs and ship with Shiprocket
+                                </div>
+                                <div class="content-group">
+                                    <div id="sale_delivery_shiprocket"/>
+                                </div>
+                            </div>
+                        </div>
```

Comment:
Don't put that in sale. It's only used in India

Reply:
Removed!